drivers/SmartThings/zigbee-window-treatment/src/test/test_zigbee_window_treatment_screen_innovations.lua

```diff
@@ -0,0 +1,260 @@
+    "WindowShade pause cmd test case",
+    function()
+      test.socket.capability:__queue_receive(
+          {
+            mock_device.id,
+            { capability = "windowShade", component = "main", command = "pause", args = {} }
+          }
+      )
+      test.socket.zigbee:__expect_send({
+        mock_device.id,
+        WindowCovering.server.commands.Stop(mock_device)
+      })
+      test.wait_for_events()
+    end
+)
+
+test.register_coroutine_test(
+    "Battery Percentage Remaining test cases",
+    function()
+      mock_device:set_field("motorState", "idle")
+      local battery_test_map = {
+          [200] = 100,
+          [100] = 50,
+          [0] = 0
+      }
+      for bat_perc_rem, batt_perc_out in pairs(battery_test_map) do
+        test.socket.zigbee:__queue_receive({ mock_device.id,
+          PowerConfiguration.attributes.BatteryPercentageRemaining:build_test_attr_report(mock_device, bat_perc_rem) })
+        test.socket.capability:__expect_send( mock_device:generate_test_message("main", capabilities.battery.battery(batt_perc_out)) )
+        test.wait_for_events()
+      end
+    end
+)
+
+test.register_coroutine_test(
+  "Refresh should generate expected messages",
+  function()
+    test.socket.zigbee:__set_channel_ordering("relaxed")
+    test.socket.capability:__queue_receive({ mock_device.id, { capability = "refresh", component = "main", command = "refresh", args = {} } })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      WindowCovering.attributes.CurrentPositionLiftPercentage:read(mock_device)
+    })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      PowerConfiguration.attributes.BatteryPercentageRemaining:read(mock_device)
+    })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      Basic.attributes.PowerSource:read(mock_device)
+    })
+  end
+)
+
+test.register_coroutine_test(
+  "doConfigure should generate expected messages",
+  function()
+    test.socket.device_lifecycle:__queue_receive({ mock_device.id, "doConfigure" })
+    test.socket.zigbee:__set_channel_ordering("relaxed")
+    mock_device:expect_metadata_update({ provisioning_state = "PROVISIONED" })
+    test.timer.__create_and_queue_test_time_advance_timer(1, "oneshot")
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      zigbee_test_utils.build_bind_request(mock_device, zigbee_test_utils.mock_hub_eui, PowerConfiguration.ID)
+    })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      PowerConfiguration.attributes.BatteryPercentageRemaining:configure_reporting(mock_device, 30, 21600, 1)
+    })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      zigbee_test_utils.build_bind_request(mock_device, zigbee_test_utils.mock_hub_eui, WindowCovering.ID)
+    })
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      WindowCovering.attributes.CurrentPositionLiftPercentage:configure_reporting(mock_device, 0, 600, 1)
+    })
```

Comment:
The reason you have both of these twice is because of the `device:configure` call which uses our default configurations for these attributes.

I'd recommend you either remove `device:configure` or use it and remove your separate bindings/configurations of these two attributes.

Reply:
Hi @greens,
Removed the `device:configure` call.